src/SAML2/SOAPClient.php

```diff
@@ -113,19 +145,20 @@ public function send(Message $msg, SimpleSAML_Configuration $srcMetadata, Simple
 
         $x = new \SoapClient(null, $options);
 
-        // Add soap-envelopes
+        /* Add soap-envelopes */
         $request = $msg->toSignedXML();
-        $request = self::START_SOAP_ENVELOPE . $request->ownerDocument->saveXML($request) . self::END_SOAP_ENVELOPE;
+        $request = self::START_SOAP_ENVELOPE.$request->ownerDocument->saveXML($request).self::END_SOAP_ENVELOPE;
 
         Utils::getContainer()->debugMessage($request, 'out');
 
-        $action = 'http://www.oasis-open.org/committees/security';
-        $version = '1.1';
```

Comment:
Any reason these are removed? They're still referenced on line 160, and it looks like they could replace the SOAPAction header added in the new code.